janusgraph-dist/src/assembly/descriptor/configs.component.xml

```diff
@@ -1,9 +0,0 @@
-<component xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/component/1.1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/component/1.1.2 http://maven.apache.org/xsd/component-1.1.2.xsd">
-    <fileSets>
-        <fileSet>
-            <directory>${assembly.static.dir}/conf</directory>
-            <outputDirectory>/conf</outputDirectory>
-        </fileSet>
-    </fileSets>
-</component>
```

Comment:
I am not very familiar with `janusgraph-dist` but why do we remove this file?

Reply:
This file is unused also without this PR.